### PR TITLE
Version 1.15.0

### PR DESCRIPTION
**Pull request by jlstevens**
This PR is still WIP and will update the CHANGELOG and releases.rst for the upcoming 1.15.0 release.

- [x] There are still a few outstanding PRs I need to review before the list of complete
- [x] There are some new PRs merged since this set of PRs was computed that also need review.
- [x] Update `releases.rst`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,169 @@
+Version 1.15.0
+==============
+**July 6, 2022**
+
+This is a major release with a large number of new features and bug
+fixes, as well as a updates to Python and Panel compatibility.
```
Comment (jbednar) [inline CHANGELOG.md:6]:
```suggestion
fixes, as well as updates to Python and Panel compatibility.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,169 @@
...
+- Added `pixel_ratio parameter` to ResamplingOperation base class
+  ([#5288](https://github.com/holoviz/holoviews/pull/5288))
+- Expose `rescale_discrete_levels` in the Bokeh backend
+  ([#5312](https://github.com/holoviz/holoviews/pull/5312))
+- Set `rescale_discrete_levels=True` by default
+  ([#5268](https://github.com/holoviz/holoviews/pull/5268))
+
+Linked selection features:
+
+The following features have been added to linked selections.
```
Comment (jbednar) [inline CHANGELOG.md:61]:
```suggestion
New linked selection features:
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,169 @@
...
+- Allow passing custom `selection_expr` to linked selections filter
+  ([#5012](https://github.com/holoviz/holoviews/pull/5012))
+- Implement linked selection lasso for cudf using cuspatial
+  ([#5051](https://github.com/holoviz/holoviews/pull/5051))
+- Fix `AdjointLayout` in `link_selections`
+  ([#5030](https://github.com/holoviz/holoviews/pull/5030))
+
+Bokeh features:
+
+The following features have been added to the Bokeh plotting backend:
```
Comment (jbednar) [inline CHANGELOG.md:74]:
```suggestion
New features for the Bokeh plotting backend:
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,169 @@
...
+- Optimize lasso selection by applying box-select first
+ ([#5061](https://github.com/holoviz/holoviews/pull/5061))
+ https://github.com/holoviz/holoviews/pull/5061
+- Support ibis-framework version 3
+ ([#5292](https://github.com/holoviz/holoviews/pull/5292))
+
+Bug fixes:
+
+In addition to the new features and enhancements, a large number of bug
+fixes have been made:
```
Comment (jbednar) [inline CHANGELOG.md:104]:
```suggestion
Bug fixes:
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,169 @@
...
+@stas-sl @Yura52 for their contributions. In addition, thanks to the
+maintainers @jbednar, @maximlt, @jlstevens and @philippjfr for
+contributing to this release.
+
+Compatibility:
+
+- Python 2 support has finally been dropped with 1.14.9 as the last
+  release supporting Python 2.
+- HoloViews now requires panel >0.13.1
+  ([#4329](https://github.com/holoviz/holoviews/pull/4329))
```
Comment (jbednar) [inline CHANGELOG.md:23]: - Compatibility note: colormaps for the output of the `datashade` operation have changed to address https://github.com/holoviz/datashader/issues/357 ; see rescale_discrete_levels below.

**Review by jbednar — APPROVED** [review]
Looks good!

File: CHANGELOG.md (hunk 5 above)
Comment (jlstevens) [reply CHANGELOG.md:23]: Good suggestion, and it may be worth having a line indicating how to switch `rescale_discrete_levels` back off

**Comment by philippjfr** [comment]
My feeling is that quite a lot of the mentioned features were backported in some form but I didn't check to confirm.

**Comment by Hoxbro** [comment]
I just checked which issues were mentioned before and got `[5051, 5063, 5211, 5288, 5290]`. 

<details>
  <summary>code </summary>

``` python
import re
import numpy as np

all_issues = list(map(int, re.findall("#(\d+)", all_releases)))
latest_issue = list(map(int, re.findall("#(\d+)", latest_release)))
doubles = np.arange(np.max(all_issues) + 1)[np.bincount(all_issues) > 1]
sorted(set(latest_issue) & set(doubles))

```

Where `all_releases` string of the full release notes and `latest_release` is the latest one. 


</details>

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,174 @@
...
+- Don't skip each renderer's `load_nb call` when multiple extension
+ calls are made in a single cell
+ ([#5302](https://github.com/holoviz/holoviews/pull/5302))
+- Set plotly range correctly for log axis
+ ([#5272](https://github.com/holoviz/holoviews/pull/5272))
+- Sanitize uses of `contextlib.contextmanager`
+ ([#5018](https://github.com/holoviz/holoviews/pull/5018))
+- Ensure `overlay_aggregate` is not applied for anti-aliased lines
+ ([#5266](https://github.com/holoviz/holoviews/pull/5266))
+ https://github.com/holoviz/holoviews/pull/5266
```
Comment (Hoxbro) [inline CHANGELOG.md:120]:
```suggestion

```

Loose url

**Comment by philippjfr** [comment]
Thanks @Hoxbro, we may also have been inconsistent about referencing the PR rather than the issue before so it's a painful problem.

**Comment by jlstevens** [comment]
@Hoxbro thanks for doing that!

I used a manual process to try and eliminate duplicates so thanks for catching the ones I missed. I'll apply your suggestion and remove the five duplicate references you found.

File: CHANGELOG.md (hunk 5 above)
Comment (jlstevens) [reply CHANGELOG.md]: Addressed in 74942a6cf

**Comment by jlstevens** [comment]
@philippjfr Please go ahead and merge if it all looks good to you now. Then we can tag a final rc and release!

**Comment by Yura52** [comment]
Regarding the updated Sankey algorithm: users may need to update the `node_padding` parameter if they have Sankey plots built with HoloViews < 1.15.0. For details, see [this comment](https://github.com/holoviz/holoviews/issues/5079#issuecomment-916162818). Mentioning this because it is relevant to the "compatibility" aspect in some sense :)

**Pull request merged**